package.json

```diff
@@ -26,7 +26,7 @@
   "scripts": {
     "test": "pnpm test:smoke lint && pnpm test:js && pnpm test:smoke",
     "test:smoke": "testem ci",
-    "test:js": "qunit",
+    "test:js": "qunit test/retry.js && qunit test/retry-only.js",
```

Comment:
Is this needed, `qunit` by default should run all tests matching `test/**/*.js`

Reply:
Yes, it's necessary! We need to separate tests using `retry.only`, if we run them together with other tests, all the rest will be skipped. Is there a better way to isolate these two runs?

Reply:
Ha, of course!